package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "To do list",
+  "name": "Books store",
```

Comment:
You might want to rename the app name to `book-store`

Reply:
I've changed the name to book-store